libnetwork/drivers/overlay/peerdb.go

```diff
@@ -221,63 +222,57 @@ func (d *driver) peerDbDelete(nid, eid string, peerIP net.IPNet, peerMac net.Har
 // networkDB has already delivered some events of peers already available on remote nodes,
 // these peers are saved into the peerDB and this function is used to properly configure
 // the network sandbox with all those peers that got previously notified.
-// Note also that this method sends a single message on the channel and the go routine on the
-// other side, will atomically loop on the whole table of peers and will program their state
-// in one single atomic operation. This is fundamental to guarantee consistency, and avoid that
+// Note also that this method will atomically loop on the whole table of peers
+// and will program their state in one single atomic operation.
+// This is fundamental to guarantee consistency, and avoid that
```

Comment:
Nit (but existing); avoid future tense;

```suggestion
// Note also that this method atomically loops the whole table of peers
// and programs their state in one single atomic operation.
// This is fundamental to guarantee consistency, and avoid that
```